apps/vc-api/test/vc-api/exchanges/consent-and-resident-card-credentialexchange/resident-card-presentation.exchange.ts

```diff
@@ -43,10 +44,25 @@ export class ResidentCardPresentation {
             {
               presentationDefinition: {
                 id: '286bc1e0-f1bd-488a-a873-8d71be3c690e',
+                submission_requirements: [
+                  {
+                    name: 'Application requirements',
+                    purpose: 'Application requires user satisfies certain criteries',
+                    rule: Rules.All,
+                    from: 'A'
+                  },
```

Comment:
@JGiter How does this execute with referent to VCs in the VP. So `group: ['A']` has one constraint and when it is validated for the two VCs (consent and resident card), does it gets executed against both? Or just against one? 

Or is it that off all execution any one constraint successful will result in a valid verification?

Reply:
I guess that this is correct
> Or is it that off all execution any one constraint successful will result in a valid verification?

so that `rule: Rule.All` refers to input_descriptors

Reply:
> so that rule: Rule.All refers to input_descriptors

In my understanding this is not correct, I think this is for the number of successful execution. If it was for `input_descriptors` then it would fail the moment it is executed against the odd. For our case `$.input_descriptors[0]` for `ConsentCredentials` and `$.input_descriptors[1]` for `PermanentResidentCard`. What do you think?

Reply:
Aah! make sense. Thanks @JGiter